src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginStatus.cshtml

```diff
@@ -2,13 +2,20 @@
 @model ActiveLogin.Authentication.BankId.AspNetCore.Areas.BankIdAuthentication.Models.BankIdLoginViewModel
 @inject IStringLocalizer<ActiveLogin.Authentication.BankId.AspNetCore.BankIdAuthenticationHandler> Localizer
 
-<div class="card" id="bankIdLoginStatus" style="display: @(Model.AutoLogin ? "block" : "none");">
-    <div class="card-body">
-        <div class="status-spinner">
-            <partial name="_spinner" />
+
+<div id="bankIdLoginStatus" style="display: @(Model.AutoLogin ? "block" : "none");">
+    <div class="card">
+        <div class="card-body">
+            <div class="status-spinner">
+                <partial name="_spinner" />
+            </div>
+            <div class="status-message">@Model.LoginScriptOptions.InitialStatusMessage</div>
         </div>
-        <div class="status-message">@Model.LoginScriptOptions.InitialStatusMessage</div>
     </div>
+
+    <button type="button" class="btn btn-danger btn-block btn-cancel mt-2">
```

Comment:
I'm not 100% sure, but I think we should use `btn-link` instead of `btn-danger`. Danger feels so... dangerous :)
I had a look at other sites, like Handelsbanken, and they do the "Link" style.

With `.btn-link`:
![A](https://user-images.githubusercontent.com/1662918/67668967-3fe36a80-f971-11e9-9464-4e54e03a1bd6.PNG)

With `.btn-danger`:
![B](https://user-images.githubusercontent.com/1662918/67668974-4245c480-f971-11e9-9dc2-d874c3243be9.PNG)

Reply:
I completely agree.